backend/api/handlers/semester/PostSemesterHandler.go

```diff
@@ -0,0 +1,54 @@
+package semester
+
+import (
+	"fmt"
+	"net/http"
+
+	"LiteracyLink.com/backend/api/model"
+	"LiteracyLink.com/backend/db"
+	"github.com/gin-gonic/gin"
+	"gorm.io/gorm"
+)
+
+func PostSemesterHandler(c *gin.Context) {
+
+	var semest model.Semester
+
+	// Bind the request body to the Semester struct
+	if err := c.ShouldBindJSON(&semest); err != nil {
```

Comment:
There might be issues binding to go time type and the []byte might pose some issues. Have we tested it?

Reply:
I have not tested it yet